pootle/static/js/shared/components/KeyboardHelpDialog.js

```diff
@@ -11,14 +11,8 @@ import { showModal } from './Modal';
 
 import './KeyboardHelpDialog.css';
 
-let keyboardHelpOpen = false;
 
 export function showKeyboardHelpDialog() {
-  if (keyboardHelpOpen) {
-    return;
```

Comment:
I checked last night and couldn't see it happening. I double-checked now and I can see it. I'll close the PR as invalid.